zerver/lib/api_test_helpers.py

```diff
@@ -468,6 +468,15 @@ def upload_file(client):
     test_against_fixture(result, fixture, check_if_equal=['msg', 'result'],
                          check_if_exists=['uri'])
 
+def get_stream_topics(client, stream_id):
+    # type: (Client, int) -> None
+
+    # {code_example|start}
+    result = client.get_stream_topics(stream_id)
+    # {code_example|end}
+
+    validate_against_openapi_schema(result, '/users/me/{stream_id}', 'get')
```

Comment:
isn't "topics" part of the URL here?

Reply:
@YagoGG what was the resolution here?  It looks like you changed this and it's now correct, but what can we do to make this sort of mistake fail tests next time?